src/containers/EditDonor/index.js

```diff
@@ -0,0 +1,145 @@
+import React, {Component, useState} from "react";
+import Button from "react-bootstrap/Button";
+import FormGroup from "react-bootstrap/FormGroup";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Card from "react-bootstrap/Card";
+import * as actions from "./actions";
+import FormControl from "react-bootstrap/FormControl";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({
+    getDonor: actions.getDonor,
+  }, dispatch);
+}
+
+export class EditDonor extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      data: {},
+      id: "",
+      name: "",
+      phone: "",
+      email: "",
+      address: "",
+      city: "",
+      state: "",
+      zipCode: "",
+      setEmail: "",
+      password: "",
+      setPassword: ""
+    }
+  }
+
+  componentDidMount() {
+    this.props.getDonor(this.props.match.params.donorId).then(response => {
+      this.setState({
+        data: response.payload,
+      })
+    })
+  }
+
+
+
+  // validateForm() {
+  //   return this.email.length > 0 && this.password.length > 0;
+  // }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+    <div style={{display: "flex", justifyContent: "center"}}>
+          <Card style={{width: "48rem"}}>
+            <Card.Header as="h5">Edit Donor</Card.Header>
+            <Card.Body>
+              <div className="Login">
+                <form onSubmit={this.handleSubmit}>
+                  <Row>
+                    <Col>
+                      <Form.Control placeholder= "First Name" />
+                    </Col>
+                    <Col>
+                      <Form.Control placeholder=  {this.state.data.email != null ?
+                           (this.state.id):'Last Name'}
```

Comment:
Why are we checking this for email here? I don't think we have last name in our db scheme so I think we could leave it as is.

Reply:
I actually don't understand why we check for state data to populate placeholders.

Reply:
The schema only has Full Name, but I will want to populate both First and Last name fields with what is appropriate and currently exists under Full Name.  This is an area where functionality is not completed yet.  I want a conditional where if the info is present I will show it as regular text inputted into the field and if it is null I will use a placeholder.